Src/Base/AMReX_GpuFuse.cpp

```diff
@@ -86,12 +89,32 @@ void Fuser::Launch ()
         constexpr int nwarps_per_block = nthreads/Gpu::Device::warp_size;
         int nblocks = (ntotwarps + nwarps_per_block-1) / nwarps_per_block;
 
+        bool is_reduction = s_in_fuse_reduction_region;
+
```

Comment:
Again, a little worried about complexity of the implementation: is FuseReductionSafeGuard meant to be hidden from the users, or phased out later?

Reply:
This is not meant for the users.  Unfortunately, it is complicated for reduction.